src/styles/base/_colors.scss

```diff
@@ -41,4 +41,8 @@
   --color-light-500: #edf3fa;
   --color-light-700: #d5dae0;
   --color-light-900: #8d9094;
+
+  /* tabs */
+  --color-primary-active-tab: #73a5ff15;
```

Comment:
Hex color should be 6 characters, but here 8. Please double check values

Reply:
Here i want to have color with transparent. And for color we decide to use css variables